recipe/meta.yaml

```diff
@@ -1,62 +1,71 @@
-{% set version = "2.3.8" %}
+{% set version = "5.6.0" %}
 
 package:
   name: pymc
```

Comment:
could set variable name (and use here and in the url), although this is not critical.

Reply:
will do.